src/erlcloud_ddb1.erl

```diff
@@ -93,7 +93,7 @@ batch_get_item(RequestItems) ->
 -spec batch_get_item([batch_get_item_request_item()], aws_config()) -> json_return().
 batch_get_item(RequestItems, Config) ->
     Json = [{<<"RequestItems">>, [batch_get_item_request_item_json(R) || R <- RequestItems]}],
-    erlcloud_ddb_impl:request(Config, "DynamoDB_20111205.BatchGetItem", Json).
+    erlcloud_ddb_impl:request(Config, false, "DynamoDB_20111205.BatchGetItem", Json).
```

Comment:
Are all of these `false` actually required? Maybe I'm missing the _bigger picture_ here, but why not make `erlcloud_ddb_impl:request/3` use `false` by default, internally, while calling newer `erlcloud_ddb_impl:request/4` (while exposing both functions)?

Reply:
This false was due to changes in erlcloud_ddb_impl:request function. I added a new request/4 and that request/3 will just make a call to this new function as suggested by @nalundgaard.